apps/site/middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { NextResponse, NextRequest } from 'next/server'
+
+export function middleware(request: NextRequest) {
+  const { pathname } = new URL(request.url)
+  if (pathname.startsWith('/docs')) {
+    return NextResponse.redirect(
+      `https://developers.vtex.com/docs/guides/faststore${pathname.replace(
+        '/docs',
+        ''
+      )}`
+    )
+  }
+  if (pathname.startsWith('/components')) {
+    return NextResponse.redirect(
+      `https://developers.vtex.com/docs/guides/faststore${pathname
+        .replace('/components', '')
+        .replace(/\/([^\/]*)\//, '/$1-')}`
```

Comment:
what does this regex do?

Reply:
faststore.dev uses `/components/atoms/checkbox` but the developer portal uses `/atoms-checkbox`. So, this regex is responsible for update only the second `/`

Reply:
you can test here, using the preview https://faststore-site-git-redirect-site-faststore.vercel.app/components/atoms/checkbox